en/components/overlay_styling.md

```diff
@@ -0,0 +1,200 @@
+    public items = [
+        {
+            name: "Group 1",
+            header: true,
+            id: 1
+        }, { name: "Option 1", id: 100 }, { name: "Option 2", id: 101 }, { name: "Option 3", id: 102 },
+        {
+            name: "Group 2",
+            header: true,
+            id: 2
+        }, { name: "Option 1", id: 200 }, { name: "Option 2", id: 201 }, { name: "Option 3", id: 202 }
+    ];
+
+    constructor(public element: ElementRef) {
+    }
+}
+```
+
+## Styling
+
+### Including Styles
+
+In order to use the [IgniteUI Theming Library](themes/index.md), we must first import the theme `index` file in our global styles:
+
+```scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+...
+```
+Now we can make use of all of the functions exposed by the IgniteUI theme engine.
+
+### Defining a palette
+
+After we've properly imported the index file, we create a custom palette that we can use. Let's define two colors that we like and use them to build a palette with [`igx-palette`](themes/palettes.md):
+
+```scss
+$red: #ac4365;
+$purple: #5434b8;
+
+$my-palette: igx-palette($primary: $red, $secondary: $purple);
+```
+
+### Defining themes
+
+We can now define the theme using our palette. In the example, we are using `igx-drop-down`, so we need to use [`igx-drop-down-theme`]({environment:sassApiUrl}/index.html#function-igx-drop-down-theme) to generate a theme for the component:
+
+```scss
+$my-drop-down-theme: igx-drop-down-theme(
+    $palette: $my-palette
+);
+```
+You can read more about styling the drop-down [here](drop_down.md#styling)
+
+In order to style the Overlay's modal background, we also use [`igx-overlay-theme`]({environment:sassApiUrl}/index.html#function-igx-overlay-theme):
+
+```scss
+$background-purple: #5443b84a;
+
+$my-overlay-theme: igx-overlay-theme(
+    $background-color: $background-purple
+);
+```
+
+### Applying the themes
+
+The easiest way to apply our theme is to `@include` the [`igx-drop-down`]({environment:sassApiUrl}/index.html#mixin-igx-drop-down) and [`igx-overlay`]({environment:sassApiUrl}/index.html#mixin-igx-overlay) mixins in our global styles file (typically `styles.scss`):
+
+```scss
+@include igx-overlay($my-overlay-theme)
+@include igx-drop-down($my-drop-down-theme)
+```
+
+This will apply our custom themes to **all** instances of Drop Downs and to **all** modal overlays, regardless of where they are spawned from.
+
+### Scoping Drop Down Theme
+
+In order to better explain how the theme is applied to the drop-down and its items, we need to first take some time to explain how Angular's [ViewEncapsulation](https://angular.io/guide/component-styles#view-encapsulation) works. `ViewEncapsulation` takes care of our styles being scoped to a certain part (component) of our application. As stated in the official Angular documentation, we can define what `ViewEncapsulation` is used by our component. For this example, we'll discuss the default one - `ViewEncapsulation.Emulated`.
```

Comment:
Parts of this, along with the next bit about how projected items pick up styles even when moved by the service should likely be a section on it's own, **but** only after the main explanation. Maybe start with the more basic styling option: `.container { @include igx-drop-down($my-drop-down-theme) }` as that will have the same problem.

If you want to make it guide-like, merge the outlet explanation here w/ the usual `:host { ::ng-deep { } }` wrap + `outlet` to achieve the expected encapsulation.